docs/en/setup/backend/backend-storage.md

```diff
@@ -265,6 +266,28 @@ storage:
 ```
 All connection related settings including link url, username and password are in `application.yml`. The Metadata storage provider settings can refer to the configuration of **H2/MySQL** above.
 
+## PostgreSQL
+Active PostgreSQL as storage, set storage provider to **postgresql**. 
```

Comment:
Could you be clear about the postgre version? Because we selected certain client driver.

Reply:
It supports PostgreSQL 8.2 or newer, and it is the latest version when I use it.

Reply:
Then update the documentation please.